core/components/header/index.tsx

```diff
@@ -80,12 +83,7 @@ const getCartCount = async () => {
     document: GetCartCountQuery,
     variables: { cartId },
     customerAccessToken,
-    fetchOptions: {
-      cache: 'no-store',
-      next: {
-        tags: [TAGS.cart],
```

Comment:
No point tagging something that's `no-store`, right?

Reply:
It's still valid as `revalidateTag` will refetch that data without reloading the page.

Reply:
thanks for explaining, I'll revert this change